faq-bundle/src/Resources/contao/modules/ModuleFaq.php

```diff
@@ -103,6 +103,11 @@ public function getSearchablePages($arrPages, $intRoot=0, $blnIsSitemap=false)
 				{
 					while ($objItems->next())
 					{
+						if ($blnIsSitemap && $objItems->robots === 'noindex,nofollow')
```

Comment:
This change seems unrelated. Can you please elaborate?

Reply:
The description of this PR is misleading. It also contains the robots setting introduced for news and events with https://github.com/contao/contao/pull/1778.

Reply:
I see. 👍🏻